packages/gatsby-telemetry/src/telemetry.ts

```diff
@@ -215,14 +215,18 @@ export class AnalyticsTracker {
   }
 
   getGatsbyVersion(): SemVer {
-    const packageInfo = require(join(
-      process.cwd(),
-      `node_modules`,
-      `gatsby`,
-      `package.json`
-    ))
     try {
-      return packageInfo.version
+      const packageJson = join(
+        require
+          .resolve(`gatsby`)
```

Comment:
could we instead of dropping 3 last segments (`slice` below) - just require `gatsby/package.json` (or resolve that require and still read it with `fs`)?

Reply:
We do something like that for example in https://github.com/gatsbyjs/gatsby/blob/257de93a6e1caf46cbad4172ef342e928aa4b97e/packages/gatsby/src/bootstrap/load-plugins/validate.ts#L5

Reply:
Yup, that should be fine. I just edited this from what it was to the least we had to change 😅 

@jamo Do you want to push that change and test it?

Reply:
yea, I'll try that a bit later

Reply:
That seems to be working